### PR TITLE
docs(website): Publish January 2024 product newsletter

### PR DESCRIPTION
**Pull request by jamilbk**
Publish the 2024 product newsletter (email version already sent)

### CODE REVIEW

**Comment by jamilbk** [comment]
Have links coming inbound to this, so I'll publish it now and address feedback in a later PR.

File: website/src/app/blog/jan-2024-product-update/readme.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+If you're on a recent Apple or Android device, you're in luck! These platforms
+are available for beta testing today. Apply for the closed beta
+[here](/product/early-access), and we'll be in touch. We're currently
+prioritizing teams and businesses at this time, but plan to roll out invites to
+individuals and homelabbers along with Windows and Linux betas over the next few
+weeks.
+
+To beta test 1.0, you'll need one of the following:
+
+- Apple silicon or Intel Mac running **macOS 12** or later
```
Comment (bmanifold) [inline website/src/app/blog/jan-2024-product-update/readme.mdx:39]: Are we officially supporting macOS 12?  For some reason I was thinking we were only going to support 13 for now?  But, I may have missed something.

Comment (jamilbk) [reply website/src/app/blog/jan-2024-product-update/readme.mdx:39]: 12 should be working now that Split DNS isn't performed in the macOS side of things. There may still be a bug with network extension prompting for Keychain access, but the DNS should in Theory be fixed.

So far, every beta tester has been running macOS 14 so I wanted to cast a wide net and see what we get w.r.t. OS version requirements.

Comment (jamilbk) [reply website/src/app/blog/jan-2024-product-update/readme.mdx:39]: cc @AndrewDryga -- have you tried with the later clients? Are you still on 12?

Comment (AndrewDryga) [reply website/src/app/blog/jan-2024-product-update/readme.mdx:39]: @jamilbk yeah, the latest testflight have whole set of new issues:

1. System DNS resolver is not broken anymore but traffic doesn't go through the tunnel at all.

2. When I click "Disconnect and Quit" and then quickly open Firezone again I'm locked out from connecting button:
Screenshot 2024-01-11 at 11 45 14

After reboot it's fine, but I need to sign in again:
Screenshot 2024-01-11 at 11 46 25

3. UI doesn't look like it's rendered properly for me:
Screenshot 2024-01-11 at 11 41 54
Screenshot 2024-01-11 at 11 44 00

**Pull request merged**